docs/UserGuide.md

```diff
@@ -334,7 +340,9 @@ Examples:
 
 >Format: `/list`
 
-Lists all the cheatsheets in the List in a table. CheatLogs then enters sorting mode. The program continues after returning. The image below illustrated what you may see after executing the command successfully.
+Lists all the cheatsheets in the List in a table. CheatLogs then enters sorting mode. 
+In sorting mode, CheatLogs accepts a number (1-4) and will sort accordingly to the message shown in Image ???. Any other characters will exit sorting mode.
```

Comment:
is the `???` intentional?

Reply:
Erm we haven't insert the image number for each image, so i just put it "???" first, i'll replace it when all the images all finalised